### PR TITLE
Clean up and update docs and settings

### PR DESCRIPTION
**Pull request by rliebz**
This makes the following changes:
- Moves the templates to the `.github/` directory where they will actually be respected
- Updates the issue templates to better resemble GitHub's built-in options
- Removes the README, which is unhelpful to have in new projects
- Update the code of conduct to the newer version from the same source
- Clean up some formatting just how I like it

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -2,75 +2,131 @@
...
 ## Enforcement
 
 Instances of abusive, harassing, or otherwise unacceptable behavior may be
-reported by contacting the project team at support@morningconsult.com. All
-complaints will be reviewed and investigated and will result in a response that
-is deemed necessary and appropriate to the circumstances. The project team is
-obligated to maintain confidentiality with regard to the reporter of an incident.
-Further details of specific enforcement policies may be posted separately.
+reported to the community leaders responsible for enforcement at
+support@morningconsult.com.
```
Comment (alexdulin) [inline CODE_OF_CONDUCT.md]: I know this was here before, but do we want to set up a different mailing list for this? I doubt anyone who responds to this today would know what to do if someone from github emailed this address.

**Review by alexdulin — DISMISSED** [review]

File: CODE_OF_CONDUCT.md (hunk 1 above)
Comment (rliebz) [reply CODE_OF_CONDUCT.md]: Fair point. You want to set up `opensource@morningconsult.com`?

Comment (alexdulin) [reply CODE_OF_CONDUCT.md]: Works for me!

Comment (rliebz) [reply CODE_OF_CONDUCT.md]: Updated

**Review by alexdulin — APPROVED** [review]

**Pull request merged**